api/v1/authentication_types.go

```diff
@@ -100,8 +100,32 @@ type AuthenticationOIDC struct {
 	// +optional
 	// +kubebuilder:validation:Enum=Verify;InsecureSkip
 	EmailVerification *EmailVerificationType `json:"emailVerification,omitempty"`
+
+	// PromptTypes is an optional list of string values that specifies whether the identity provider prompts the end user
+	// for re-authentication and consent. See the RFC for more information on prompt types:
+	// https://openid.net/specs/openid-connect-core-1_0.html. The defined values are:
+	// ["None", "Login", "Consent", "SelectAccount"].
```

Comment:
Minor: Since you already explain the allowable values and the RFC further down with the PromptType declaration, you don't need to also explain it here. 

I think the first sentence you have adequate, though it may be worth mentioning what the default behaviour is (if the user does not define PromptTypes).